llama_index/vector_stores/elasticsearch.py

```diff
@@ -21,6 +23,8 @@
     "EUCLIDEAN_DISTANCE",
 ]
 
+nest_asyncio.apply()
```

Comment:
Perhaps we should move this `.apply()` into `ElasticsearchStore.__init__`? There's some minor tradeoffs associated with `nest_asyncio` so should only apply it imo if they've actually instantiated this class

Reply:
Sure, that makes sense.